src/jit/SimdX86Inl.h

```diff
@@ -58,6 +60,10 @@ enum Type : uint32_t {
     cvtps2pd = 0x5a,
     divpd = 0x5e | SimdOp::prefix66,
     divps = 0x5e,
+    maxpd = 0x5f | SimdOp::prefix66,
```

Comment:
Some of these have been already defined, please rebase the patch to the latest code.